libqtile/widget/statusnotifier.py

```diff
@@ -94,7 +94,23 @@ def __eq__(self, other):
 
     async def start(self):
         # Create a proxy object connecting for the item.
-        introspection = await self.bus.introspect(self.service, self.path)
+
+        # Some apps provide the incorrect path to the StatusNotifier object
+        # We can try falling back to the default if that fails.
+        # See: https://github.com/qtile/qtile/issues/3418
+        found_path = False
+
+        while not found_path:
+            try:
+                introspection = await self.bus.introspect(self.service, self.path)
+                found_path = True
+            except InvalidObjectPathError:
+                logger.info(f"Cannot find {self.path} path on {self.service}.")
+                if self.path == STATUSNOTIFIER_PATH:
+                    return False
+
+                # Try the default ('/StatusNotifierItem')
+                self.path = STATUSNOTIFIER_PATH
```

Comment:
Does this make a second attempt only? If so, it might be more clear to read by making the second attempt explicit rather than using a `while` loop.

Reply:
It is a max of two tries but I did it like this to prevent duplication of code and nested try... except... blocks.

Reply:
OK, fair. Could you add a comment to it just to say that it's retrying once and then using the default path?

Reply:
> 

Sure. That's now in lines https://github.com/qtile/qtile/pull/3421/files#diff-fe63051e6776710337802714148ada06e53b85646a48102a0444826aa5df4287R97-R101